src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -390,18 +390,18 @@ public async Task<IDictionary<string, WorkerStatus>> GetWorkerStatusesAsync()
 
         public async Task ShutdownAsync()
         {
-            _logger.LogDebug($"Waiting for {nameof(RpcFunctionInvocationDispatcher)} to shutdown");
+            _logger.LogDebug("Waiting for {name} to shutdown", nameof(RpcFunctionInvocationDispatcher));
```

Comment:
This is effectively a constant, shouldn't be structured. Same with lines 400 and 404